cloud_functions/create_questionnaire_case_tasks.py

```diff
@@ -37,9 +37,20 @@ def retrieve_world_ids(config: Config, filtered_cases: List[Dict[str, str]]) ->
     )
     worlds = optimise_client.get_worlds()
 
-    world_map_with_world_ids = {world["identity"]["reference"]: world["id"] for world in worlds}
-    return [world_map_with_world_ids[case["qDataBag.FieldRegion"]] for case in filtered_cases]
-
+    world_map_with_world_ids = {world["identity"]["reference"]: world["id"] for world in worlds}    
+    # return [world_map_with_world_ids[case["qDataBag.FieldRegion"]] for case in filtered_cases]
+    
+    new_filtered_cases = []
+    world_ids = []
+    for case in filtered_cases:
+        if case['qDataBag.FieldRegion'] == "":
```

Comment:
I think this should actually be if it's missing (i.e. `if 'qDataBag.FieldRegion' not in case`)

Reply:
@Riceyo said it should be empty value so == ""